### ISSUE TITLE
Language Changes

### ISSUE THREAD

**Issue by chipallen2** [tldraw/tldraw-beta#71]
New tldraw.com site keeps changing me back to russian everytime i make a new project.

File > New Shared Project 
It opens in russian 
Then I have to click down in bottom right to help icon > english.

**Comment by TodePond** [comment]
Thanks for letting us know! The UI defaults to your browser language, so you could change your browser's language to English if you want it to default to English. We should make it stick with your choice better though 👍

**Comment by chipallen2** [comment]
You are right thanks. Changed default on mac to English and now navigator.language seems right and tldraw thinks I want english.

But still there was a bug - my language was set to Italian - not russian.

**Comment by chipallen2** [comment]
Nevermind still a bug - I just refreshed again - now back to russian.

I checked my console - my language is still english.

![image](https://user-images.githubusercontent.com/4950543/231554908-e5f71046-7bb1-4e05-9e25-54f4c88323b7.png)

**Comment by chipallen2** [comment]
Happens when I click File > New Shared Project
Other actions like file > open or refresh page do not cause it.

**Comment by TodePond** [comment]
Hey sorry about that. Currently, the language choice is set per project. So if it was originally set to Russian, it will stay in Russian unless you change it.

Soon we're going to improve this so your preferences go over to new projects, but it's not ready yet
